tools/tertiary-analysis/sc3/sc3-sc3-prepare.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="sc3-prepare" name="SC3 read scater object" version="@TOOL_VERSION@+galaxy0">
+  <description>Loads normalised Scater object into a SC3 object</description>
+  <macros>
+    <import>sc3-macros.xml</import>
+  </macros>
+  <expand macro="requirements" />
+  <command detect_errors="exit_code"><![CDATA[
+  ln -s '$scater_log' 'scater_log.rds'; 
```

Comment:
Is this line obsolete?

Reply:
Hi @nh3 , I think I created that one when I started making the wrapper using planemo so it could work in my machine. Its fixed now.

Reply:
It is working now after applying your suggestion from the comment below.